docs/UserGuide.md

```diff
@@ -1,12 +1,11 @@
 # User Guide
-  ```
- _       _      ____  __       __
-| |\ \  | |    / /\ \ \ \     / /
-| | \ \ | |   / /__\ \ \ \   / /
-| |  \ \| |  / /----\ \ \ \ / /
-|_|   \ \_| / /      \ \ \_V_/   @NUS 
-```
+![Logo](UG_Figures/Nav@NUSLogo.jpg)
+## Opening words
+Welcome to Nav@NUS application's user guide! <br><br>
+The purpose of this user guide is to provide you with all the necessary information to use this application to navigate
+around NUS's Kent Ridge Campus via the school's shuttle service.<br>
```

Comment:
I think the bus routes have BTC bus stops oso?

Reply:
Dine options also include BTC campus?

Reply:
Yah

Reply:
Alright will change all mentions of Kent Ridge Campus to NUS Campus